Add device state limitations in Overkiz (#36791)
home-assistant/home-assistant.github.io: source/_integrations/overkiz.markdown

### Overkiz API limits

**Device state changes are not broadcasted for all devices**

> Some Overkiz devices do not broadcast status changes. To update their status, the vendor's app (e.g., Somfy TaHoma) requests a status update when opened, which then broadcasts the states via events that the Overkiz integration also listens to. The Overkiz integration cannot replicate this behavior, as it does not know when you access the Home Assistant dashboard or run automations.

Reviewer: ```suggestion
Some Overkiz devices do not broadcast status changes. To update their status, the vendor's app (for example, Somfy TaHoma) requests a status update when opened. The app then broadcasts the states via events that the Overkiz integration also listens to. The Overkiz integration cannot replicate this behavior, as it does not know when you access the Home Assistant dashboard or run automations.
```
